assets/js/cookiebar-advanced.js

```diff
@@ -1,22 +1,22 @@
 /**
  * @file JS file for the cookiebar module
  *
- * @author KeesTM <developers@kees-tm.nl>
- * @author Tom Grootjans <tom@kees-tm.nl>
+ * @author EstDigital <developers@estdigital.nl>
+ * @author Tom Grootjans <tom@estdigital.nl>
  */
 
 /* Set default variables */
-var keesCookieName = 'CookieConsent';
+var CookieName = 'CookieConsent';
```

Comment:
keesCookieName zou cookieName met een kleine 'c' aan t begin

Reply:
Dit. Hetzelfde voor de vars hieronder in de drupalSettings